modal/runner/shared/common.py

```diff
@@ -7,6 +7,6 @@
     api_key_id="RUNNER_API_KEY",
 )
 
-BACKLOG_THRESHOLD = 30
+BACKLOG_THRESHOLD = 100
```

Comment:
moving this up higher now that we (should?) have some more resource isolation between containers